ddtrace/tracer/tracer.go

```diff
@@ -130,6 +132,13 @@ const payloadQueueSize = 1000
 
 func newUnstartedTracer(opts ...StartOption) *tracer {
 	c := newConfig(opts...)
+	envRules, err := samplingRulesFromEnv()
+	if err != nil {
+		log.Warn("Error(s) reading sampling rules: %s", err)
```

Comment:
This is going to look like:
```
WARN Error(s) reading sampling rules: error(s) parsing DD_TRACE_SAMPLING_RULES: rate not provided invalid rate: abc
```
Repetitive, slightly badly formatted and confusing. We can do better.